content/en/lotus/developers/glif-nodes.md

```diff
@@ -88,7 +92,31 @@ You can test that it works with:
 lotus net id 12D3KooWBF8cpp65hp2u9LK5mh19x67ftAam84z9LsfaquTDSBpt
 ```
 
-If the above does not work, verify that you are using the right token and multiaddress.
+### Using with Lotus lite node
+
+If you want to run a Lotus lite node connected to Glif endpoints, there are some specific considerations:
+
+{{< alert icon="warning" >}}
+When using Lotus v1.33.0 or later with lite nodes, you may encounter WebSocket errors because the lite node assumes the full node supports RPC v2, which Glif nodes do not currently support.
+{{< /alert >}}
```

Comment:
Is there a way to work around these errors?

Also, given 1.33.0 has been out for awhile, is there any reason not to just assume that users will hit this case?

Reply:
Removed it in: https://github.com/filecoin-project/lotus-docs/pull/804/commits/7a45cdc28b4f9babd3d6365585ec88229a66534d